server/api/controllers/baptisms/create-baptism.js

```diff
@@ -0,0 +1,65 @@
+module.exports = {
+  friendlyName: "Create baptisms",
+
+  description: "Create a baptism record",
+
+  inputs: {
+    officialName: {
+      type: "string",
+    },
+    userId: {
+      type: "string",
+    },
+    classAttendance: {
+      type: "json",
+    },
+    baptismDate: {
+      type: "string",
+    },
+    baptismPlace: {
+      type: "string",
+    },
+  },
+
+  exits: {
+    success: {
+      description: "Baptism record created successfully",
+    },
+    invalid: {
+      responseType: "badRequest",
+      description: "Failed to create baptism record",
+    },
+
+    duplicateRecords: {
```

Comment:
Same as the other file, is it being used?

Reply:
not being used